### DISCUSSION TITLE
Sensors

### DISCUSSION THREAD

**Discussion by calvaradESU** [opendcim/openDCIM#1442] [Q&A] [answered]
I have some small UPS's that have an ambient temp sensor.  I have tried setting up a sensor template for them in opendcim, but even though I can snmpget the OID's successfully from the server, when I look at a device using that template, the "Sensor Readings" section shows Last Read: Error, and temp and humidity both show zero.

The Poll tag is set on the device.

These devices don't have a humidity sensor, so I'm wondering if I hit the 3 error polling limit, since the humidity OID was not initially defined in my template.  I have since added a humidity OID which returns 0 instead of an error on snmpget.

Anybody have any ideas how I can make this work?  How can I restart polling if I hit the 3 error limit?  If that's not the problem, where would I look for errors?  There's nothing in the apache error log.

**Comment by timlegge** [comment, marked as answer]
You need to run the polling job via cron - see scheduled jobs in the wiki I
believe...

Timothy Legge
***@***.***
***@***.***
